packages/core/contracts-ovm/insured-bridge/implementation/OVM_BridgeDepositBox.sol

```diff
@@ -118,44 +119,50 @@ contract OVM_BridgeDepositBox is OVM_CrossDomainEnabled, OVM_Testable {
      **************************************/
 
     /**
-     * @notice Changes the L1 withdraw associated with this L2 deposit box.
-     * @dev Only callable by the existing bridgeRouter via the optimism cross domain messenger.
-     * @param _bridgeRouter address of the new L1 withdrawContract.
+     * @notice Changes the L1 administrator associated with this L2 deposit box.
+     * @dev Only callable by the existing bridgeAdmin via the optimism cross domain messenger.
+     * @param _bridgeAdmin address of the new L1 admin contract.
      */
-    function setBridgeRouter(address _bridgeRouter) public onlyFromCrossDomainAccount(bridgeRouter) {
-        _setBridgeRouter(_bridgeRouter);
+    function setBridgeAdmin(address _bridgeAdmin) public onlyFromCrossDomainAccount(bridgeAdmin) {
+        _setBridgeAdmin(_bridgeAdmin);
     }
 
     /**
      * @notice Changes the minimum time in seconds that must elapse between withdraws from L2->L1.
-     * @dev Only callable by the existing bridgeRouter via the optimism cross domain messenger.
-     * @param _MinimumBridgingDelay the new minimum delay.
+     * @dev Only callable by the existing bridgeAdmin via the optimism cross domain messenger.
+     * @param _minimumBridgingDelay the new minimum delay.
      */
-    function setMinimumBridgingDelay(uint64 _MinimumBridgingDelay) public onlyFromCrossDomainAccount(bridgeRouter) {
-        _setMinimumBridgingDelay(_MinimumBridgingDelay);
+    function setMinimumBridgingDelay(uint64 _minimumBridgingDelay) public onlyFromCrossDomainAccount(bridgeAdmin) {
+        _setMinimumBridgingDelay(_minimumBridgingDelay);
     }
 
     /**
      * @notice Enables L1 owner to whitelist a L1 Token <-> L2 Token pair for bridging.
-     * @dev Only callable by the existing bridgeRouter via the optimism cross domain messenger.
+     * @dev Only callable by the existing bridgeAdmin via the optimism cross domain messenger.
      * @param l1Token Address of the canonical L1 token. This is the token users will receive on Ethereum.
      * @param l2Token Address of the L2 token representation. This is the token users would deposit on optimism.
+     * @param l1BridgePool Address of the L1 withdrawal pool linked to this L2+L1 token.
      */
-    function whitelistToken(address l1Token, address l2Token) public onlyFromCrossDomainAccount(bridgeRouter) {
+    function whitelistToken(
+        address l1Token,
+        address l2Token,
+        address l1BridgePool
+    ) public onlyFromCrossDomainAccount(bridgeAdmin) {
         whitelistedTokens[l2Token] = L2TokenRelationships({
             l1Token: l1Token,
+            l1BridgePool: l1BridgePool,
```

Comment:
whitelisted token mapping now needs to remember each pool associated with each L1 token.

Reply:
great point. this is key when actually pulling out the funds onto L1.